src/PlaneWaveBasis.jl

```diff
@@ -102,7 +102,7 @@ Base.eltype(basis::PlaneWaveBasis{T}) where {T} = T
         sizehint!(mapping, n_guess)
         sizehint!(Gvecs_k, n_guess)
```

Comment:
What about the size hint for non-variational. Should be prod(fft_size) no?

Reply:
Right now this is a hack, for non-variational there's better things to do than what we do now. The main use case is still (for now at least...) variational=true, so let's not bother optimizing for this